[2023-03-13 21:11:31,874] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: test.create_table manual__2023-03-13T21:11:30.080490+00:00 [queued]>
[2023-03-13 21:11:31,904] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: test.create_table manual__2023-03-13T21:11:30.080490+00:00 [queued]>
[2023-03-13 21:11:31,905] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-03-13 21:11:31,906] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-03-13 21:11:31,908] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-03-13 21:11:31,922] {taskinstance.py:1377} INFO - Executing <Task(PostgresOperator): create_table> on 2023-03-13 21:11:30.080490+00:00
[2023-03-13 21:11:31,933] {standard_task_runner.py:52} INFO - Started process 4299 to run task
[2023-03-13 21:11:31,941] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'test', 'create_table', 'manual__2023-03-13T21:11:30.080490+00:00', '--job-id', '87', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpsojz2d8l', '--error-file', '/tmp/tmpej03nywc']
[2023-03-13 21:11:31,942] {standard_task_runner.py:80} INFO - Job 87: Subtask create_table
[2023-03-13 21:11:32,091] {task_command.py:369} INFO - Running <TaskInstance: test.create_table manual__2023-03-13T21:11:30.080490+00:00 [running]> on host 5282528dd996
[2023-03-13 21:11:32,254] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=test
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2023-03-13T21:11:30.080490+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-13T21:11:30.080490+00:00
[2023-03-13 21:11:32,294] {base.py:68} INFO - Using connection ID 'postgres' for task execution.
[2023-03-13 21:11:32,307] {dbapi.py:208} INFO - Running statement: 
            create table if not exists users(
                firstname TEXT NOT NULL,
                lastname TEXT NOT NULL,
                country TEXT NOT NULL,
                username TEXT NOT NULL,
                password TEXT NOT NULL,
                email TEXT NOT NULL
            );
        , parameters: None
[2023-03-13 21:11:32,309] {postgres.py:94} INFO - NOTICE:  relation "users" already exists, skipping

[2023-03-13 21:11:32,326] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=test, task_id=create_table, execution_date=20230313T211130, start_date=20230313T211131, end_date=20230313T211132
[2023-03-13 21:11:32,395] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-03-13 21:11:32,434] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
